Makefile

```diff
@@ -22,10 +22,14 @@ E2E_TESTS ?= ./test/e2e/*.bats
 
 E2E_PVC ?= test/e2e/resources/pvc-maven.yaml
 E2E_SECRET ?= test/e2e/resources/secret-maven.yaml
+E2E_PROXY_SECRET ?= test/e2e/resources/secret-proxy-maven.yaml                #
```

Comment:
what is the difference between these 2 secrets ???

Reply:
The difference between server_secret and proxy_secret lies in their usage and the type of information they retrieve :-
- server_secret : retrieves server access credentials.
- proxy_secret : retrieves proxy server access credentials and proxy configuration values.

A proxy server can be used to access repositories located in restricted or external networks by routing requests through it. Similar to accessing a repository server, if your network environment requires authentication to access the proxy server, you'll need to provide credentials to Maven. These credentials are also stored in the settings XML file under the <proxies> section.
In Maven, a "server" typically refers to a remote repository server from which Maven retrieves dependencies or to which it deploys artifacts.